dispatcher/MongoConnect.py

```diff
@@ -195,7 +197,8 @@ def AggregateStatus(self):
                 try:
                     rate += doc['rate']
                     buff += doc['buffer_size']
-                except:
+                except Exception as e:
+                    self.log.debug(f'Rate calculation ran into {type(e)} {e}')
```

Comment:
I think this is unnecessary. We don't care if something goes wrong while aggregating the rate because it has no impact on anything, but we don't want the program to stop.

Reply:
Bootstrax will start depending on it soon

Reply:
Bootstrax aggregates, right? It doesn't depend on a single value.

Reply:
I want to add functionality to, if the aggregation fails (e.g. because the run is older than we keep the status in the database) check this number and use it.

Reply:
Especially for testing this makes sense